airbyte-integrations/connectors/source-woocommerce/integration_tests/sample_config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "shop": "pebblepost429763178.wpcomstaging",
+  "start_date": "2020-01-01",
+  "api_key": "ck_35ddd82891a6997e10ac7c3c5e2a0f4cdd958e30",
```

Comment:
Are those credentials real? If so, they should not be here, please remove them ASAP.

Reply:
They are just for test purposes. But Thanks

Reply:
done